hera_cal/lst_stack/calibration.py

```diff
@@ -490,6 +545,38 @@ def lstbin_absolute_calibration(
                 (len(stack.nights), stack.freq_array.size, 2), dtype=complex
             )
 
+    if run_cross_pol_phase_cal and not all_nights_flagged:
+        for pol in stack.pols:
+            pol1, pol2 = utils.split_pol(pol)
+
+            if pol1 != pol2:
+                cross_pols_in_data = True
+                break
+        else:
+            cross_pols_in_data = False
+
+        if cross_pols_in_data:
+            delta = _lstbin_cross_pol_phase_calibration(stack=stack, model=model)
+            calibration_parameters["delta"] = delta
+
+            # Get antennas
+            gain_ants = set()
+            for ap in stack.antpairs:
+                gain_ants.update(ap)
+
+            gain_ants = list(gain_ants)
+
+            for ant in gain_ants:
+                for pol in unique_pols:
+                    if pol != "Jee":
```

Comment:
I don't like this hard-coded here. I'd either make this an explicit parameter (refpol) or perhaps have it follow the same logic as above about comparing pyuvdata polnums

Reply:
Fixed this with a reference polarization